setup.cfg

```diff
@@ -49,6 +49,7 @@ keywords =
 
 [options]
 install_requires =
+    sqlalchemy==1.4.46
```

Comment:
let's not do this

Reply:
replaced with `<2.0.0` Once I figure out a solution, this will be taken out.